advancedblacklist/commands/blacklist.py

```diff
@@ -0,0 +1,171 @@
+        await add_to_blacklist(self.bot, users, reason)
+        that = "that user" if len(users) == 1 else "those users"
+        await ctx.send(f"Done. Added {that} to the blacklist.")
+
+    @blacklist.command(name="remove", aliases=["del", "delete", "rm"], require_var_positional=True)
+    async def blacklist_remove(self, ctx: commands.Context, *users: discord.User):
+        """Remove users from the blacklist.
+
+        **Arguments**
+            - `users` The users to remove from the blacklist.
+        """
+        if not await get_blacklist(self.bot):
+            return await ctx.send("There are no users in the blacklist.")
+        await remove_from_blacklist(self.bot, users)
+        that = "that user" if len(users) == 1 else "those users"
+        await ctx.send(f"Done. Removed {that} from the blacklist.")
+
+    @blacklist.command(name="reason")
+    async def blacklist_reason(self, ctx: commands.Context, user: discord.Member, *, reason: str):
+        """Edit the reason for a user in the blacklist.
+
+        **Arguments**
+            - `user` The user to edit the reason of.
+        """
+        if user.bot:
+            return await ctx.send("That user is a bot.")
+        elif await self.bot.is_owner(user):
+            return await ctx.send("That user is a bot owner.")
+        elif not await in_blacklist(self.bot, user.id, ctx.guild):
+            return await ctx.send("That user is not in the blacklist.")
+        try:
+            await edit_reason(self.bot, user, reason, False)
+        except KeyError:
+            return await ctx.send("That user was not in the blacklist.")
+        await ctx.send("Done. Edited the reason for that user.")
+
+    @blacklist.command(name="list")
+    async def blacklist_list(self, ctx: commands.Context):
+        """List the users in the blacklist."""
+        if not (bl := await get_blacklist(self.bot)):
+            return await ctx.send("There are no users in the blacklist.")
+        msg = "Blacklisted Users:"
+        for key, value in bl.items():
+            name = u.name if (u := self.bot.get_user(key)) else "Unknown User"
+            msg += f"\n\t- [{key}] {name}: {value}"
+        await ctx.send_interactive(pagify(msg, page_length=1800), "yml")
+
+    @commands.group(name="localblacklist", aliases=["localblocklist"])
+    @commands.admin_or_permissions(manage_guild=True)
+    @commands.guild_only()
+    async def local_blacklist(self, ctx: commands.Context):
+        """Manage the local blacklist for your guild."""
+        pass
+
+    @local_blacklist.command(name="add")
+    async def local_blacklist_add(
+        self, ctx: commands.Context, users: commands.Greedy[discord.Member], *, reason: str = None
+    ):
+        """Add users to the local blacklist
+
+        **Arguments**
+            - `users` The users to add to the local blacklist. These cannot be bots
+            - `reason` The reason for adding these users to the blacklist. This is optional
+        """
```

Comment:
```suggestion
        self, ctx: commands.Context, members: commands.Greedy[discord.Member], *, reason: str = None
    ):
        """Add members to the local blacklist.

        **Arguments**
            - `members` The members to add to the local blacklist. These cannot be bots.
            - `reason` The reason for adding these members to the blacklist. This is optional.
        """
```
I'm always in favor of referencing users as "members" when the "context" of the command focuses on guild members instead of users. It makes it slightly easier for the end user to understand what information they are passing to the command

Reply:
Whoops, I accidentally deleted a review comment which requests for the same as this comment, across all of the blacklist commands.